vispy/app/canvas.py

```diff
@@ -48,6 +48,7 @@ class Canvas(object):
         Whether to create the widget immediately. Default True.
     vsync : bool
         Enable vertical synchronization.
+        Note for Qt backend: this setting might not work on every platform [src](https://doc.qt.io/qt-6/qsurfaceformat.html#setSwapInterval).
```

Comment:
Does this render properly in the HTML? This seems like a markdown link but most docstrings should be restructuredtext, unless we have pluguins parsing markdown too. Let's see...